tools/snapshot/snapshot.py

```diff
@@ -476,6 +481,30 @@ def is_progressed_nodes():
     return all(t.get() for t in threads)
 
 
+def page(error):
+    """
+    Send page to Pager Duty.
+    """
+    log.debug("sending pager")
+    if pager_duty['ignore']:
+        log.debug("ignoring pager-duty")
+        return
+    my_ip = ''
+    try:
+        my_ip = requests.get('http://ipecho.net/plain').content.decode().strip()
+    except Exception as e:  # catch all to page no matter what
+        log.error(f'page request machine IP error {e}')
+    trigger_response = Alert(pager_duty['service_key_v1']).trigger(
+        description=f'Snapshot failed: {error}',
+        details={
```

Comment:
Add a link to runbook on how to resolve the issue, we can add a placeholder at first though.